python/lsst/pex/config/config.py

```diff
@@ -60,6 +67,18 @@
         YamlLoaders += (CLoader,)
     except ImportError:
         pass
+else:
+    YamlLoaders = ()
+    doImport = None
+
+
+class PexGenericAlias(GenericAlias):
```

Comment:
Should have a docstring, even if the intended audience is just other maintainers.  Speaking for myself, I had to look up `types.GenericAlias` to be able guess what this was about, and I think you need say something about how it should be used and what users must do (implement `_parseTypingArgs`, I assume?  If so, a method stub with its own docs would be welcome, even if it can't be a real `abstractmethod`).

I also suspect `Pex` should be replaced with `PexConfig`; we may not have any other `pex_` packages around anymore, but they still aren't synonyms.